check_terms/src/main.rs

```diff
@@ -0,0 +1,55 @@
+use std::env;
+use std::fs::{File, OpenOptions};
+use std::io::{Read, Write};
+use std::path::Path;
+use toml;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Clone, PartialEq, Eq, Serialize)]
+struct Terms {
+    term: String,
+    translation: String,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+struct Root {
+    terms: Vec<Terms>,
+}
```

Comment:
Sorry I forgot to tell you that we have the serialization/deserialization structs, parsing and dumping APIs, and a simple formattingg script for users defined in `src/`. See 

https://github.com/EtherTW/Terms/blob/4baa03f12f6dbd17fde54db7b82703530be57dc4/src/main.rs#L6-L35

and
 https://github.com/EtherTW/Terms/blob/4baa03f12f6dbd17fde54db7b82703530be57dc4/src/main.rs#L69-L70